limitQueryAmountMiddleware.go

```diff
@@ -0,0 +1,75 @@
+package graphql
```

Comment:
Please rename file from camel case to snake case and drop the middleware:
`limit_operation_amount.go`